composer.json

```diff
@@ -13,8 +13,8 @@
   ],
   "require": {
     "cweagans/composer-patches": "^1.6",
-    "drush/drush": "^10.3 || ^11",
-    "google/apiclient": "^2.8"
+    "drush/drush": "^10.3 || ^11 || ^12",
```

Comment:
Do we require any changes in the [Drush Commands](https://github.com/acquia-pso/dst-entity-generate/tree/master/src/Commands) based on the changes in the version?